test/util/TestEntity.js

```diff
@@ -0,0 +1,16 @@
+require('require-self-ref')
```

Comment:
Actually I went back on this since it's shared on the integration and unit tests. Perhaps we could move it to `test/util/dao/TestEntity.js`, but I want to do all of the cleanup in another PR. There is some other work I want to do, but this PR is getting large.